src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py

```diff
@@ -90,6 +90,7 @@ def _stream_logs(byte_size,
     max_sleep_time = 15
     num_fails = 0
     num_fails_for_backoff = 3
+    firstTime = True
```

Comment:
Naming. Please install the python extension and fix all places where you see a warning.